utils/format/textformatter.go

```diff
@@ -33,6 +33,8 @@ func (f *TextFormatter) printItemAux(w *tabwriter.Writer, item interface{}) erro
 		showTags := strings.Split(it.Type().Field(i).Tag.Get("show"), ",")
 		if !utils.Contains(showTags, "noshow") {
 			switch it.Field(i).Type().String() {
+			case "time.Time":
```

Comment:
Not sure I understand what we are doing here

Reply:
Basically in previous PR, the print formatter required changes to support nested structures (as "Labelables" ones), then the code was re-factored and updated  in order to support it (The old code made integration and compatibility difficult).

But after that while testing a  command battery , this "time" type case was not included or not managed as it should be (so fixed here).

Probably in the future, formatter will be improve a bit more.